src/main/java/com/ericsson/ei/frontend/WebController.java

```diff
@@ -54,7 +54,9 @@ public String testRules(Model model) {
     @RequestMapping("/eiInfo.html")
     public String eiInfo(Model model, HttpServletRequest request) {
         model.addAttribute("frontendServiceUrl", frontEndUtils.getFrontEndServiceUrl());
-        model.addAttribute("frontendServiceVersion", frontEndUtils.getVersion());
+        model.addAttribute("version", frontEndUtils.getVersion());
+        model.addAttribute("enterpriseVersion", frontEndUtils.getEnterpriseVersion());
+        model.addAttribute("enterpriseVersionName", frontEndUtils.getEnterpriseVersionName());
```

Comment:
Why do we need version name?

Reply:
It is added as an optional parameter in the application.properties so the sys-admins may enter an internal version, what ever they want to call that version. if that version is left empty it will not be displayed. You where here when we discussed it.

Reply:
Parameter removed.